core/shmem.cc

```diff
@@ -0,0 +1,26 @@
+#include "core/shmem.h"
+
+namespace holodeck::core {
+
+constexpr std::string HOLODECK_BASE_PATH = "/HOLODECK_MEM";
+
+Shmem::Shmem(const std::string &name, int bufferSize,
```

Comment:
Should bufferSize be a const reference?

Reply:
It's fine in this case. As far as I understand, an int is 4 bytes, and a reference is essentially a pointer, which is 8 bytes, so it is faster to just pass small object.